packages/loader/container-loader/src/prefetchDocumentStorageService.ts

```diff
@@ -43,11 +43,11 @@ export class PrefetchDocumentStorageService extends DocumentStorageServiceProxy
             if (prefetchedBlobP !== undefined) {
                 return prefetchedBlobP;
             }
-            const prefetchedBlobPFromStorage = this.internalStorageService.read(blobId);
```

Comment:
This code no longer has intention, so it does not know if it works with binary data or string. Because of that, it needs to assume it's binary date and use readBlob() API (when pre-caching data). And when/if actual readString() comes in for this blob, we would need to treat it as utf-8 string and convert to string.
I'd personally leave this code as is. It will work correctly, but possibly inefficiently (depends on actual storage implementation). Given it's used only with r11s driver today, and r11s driver uses base64 encoding for everything, it will not add inefficiency to existing process (it would add inefficiency for SPO driver, but SPO path is not using this code).